streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/CumulativeCount.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.streams.processor.internals.metrics;
```

Comment:
Should we leave it in the `org.apache.kafka.common.metrics.stats`, if it is more generally usable? Do you intend to keep it only within streams scope? @lendle @vvcephei

Reply:
It's in here to keep it private and avoid a KIP.

Do you want to push for a KIP anyway? I think it would be pretty quick, since the change is small and well understood.

Reply:
I think it is worth to push a quick KIP, as the issue that existing `Count` which is a sampled stat may actually not be well known. If @lendle does not have time for it, @vvcephei could you help on driving such a KIP?

Reply:
Sure thing.

I might suggest just deprecating Count and adding SampledCount and CumulativeCount or something similar.